flare/gp.py

```diff
@@ -657,6 +657,20 @@ def predict(self, x_t: AtomicEnvironment, d: int) -> [float, float]:
 
         return pred_mean, pred_var
 
+    def predict_force_xyz(self, x_t: AtomicEnvironment) -> ("np.ndarray", "np.ndarray"):
+        """
+        Simple wrapper to predict all three components of a force in one go.
+        :param x_t:
+        :return:
+        """
+        forces = []
+        stds = []
+        for d in (1, 2, 3):
+            force, std = self.predict(x_t, d)
+            forces.append(force)
+            stds.append(std)
+        return np.array(forces), np.array(stds)
+
```

Comment:
This function is very useful. Probably change the `gp.predict()` to `gp.predict_force_xyz()` in predict.py, so that the loops are wrapped

Reply:
Great idea. Done!